scanpopup.hh

```diff
@@ -125,6 +124,10 @@ private:
 
   Config::Class & cfg;
   bool isScanningEnabled;
+  bool isTranslateWordFromClipboard = false;
```

Comment:
GoldenDict aims to support building with C++98, so data members have to be initialized in the constructor.

Reply:
fixed